prover/prover_fri/README.md

```diff
@@ -17,34 +17,46 @@ Machine specs:
 - RAM: 60GB of RAM(if you have lower RAM machine enable swap)
 - Disk: 400GB of free disk
 
+Before starting, make sure you go into the root of the repository, then run
+
+```
+export ZKSYNC_HOME=$(pwd)
```

Comment:
after this PR gets in: https://github.com/matter-labs/zksync-era/pull/188 we should have more clear error for users.

Reply:
nice, I think it's still ok to leave the comment though